tests/Admin/AdminTest.php

```diff
@@ -960,7 +960,7 @@ public function testGetBaseCodeRoute(): void
 
         $this->assertSame($postAdmin->getCode(), $postAdmin->getBaseCodeRoute());
 
-        $postAdmin->addChild($commentAdmin);
+        $postAdmin->addChild($commentAdmin, 'post');
```

Comment:
Should probably be backported to 3.x, right?

Reply:
Will check but it is strange that there was no error when merging that addChildren PR

Reply:
> Probably because of `$RELEVANT_FILES`